Cognite.Extensions/CdfMetrics.cs

```diff
@@ -8,8 +9,10 @@ static class CdfMetrics
             "Number and duration of asset requests to CDF", "endpoint");
         public static Summary TimeSeries { get; } = Metrics.CreateSummary("extractor_utils_cdf_timeseries_requests",
             "Number and duration of time-series requests to CDF", "endpoint");
-        public static Summary CoreTimeSeries { get; } = Metrics.CreateSummary("extractor_utils_cdf_core_timeseries_requests",
-            "Number and duration of core data model time-series requests to CDF", "endpoint");
+        public static Summary.Child Instances(ViewIdentifier view, string operation)
+        {
+            return Metrics.CreateSummary("extractor_utils_cdf_instances_requests", "Number and duration of instance requests CDF data modeling", "endpoint", "view_identifier").WithLabels(operation, $"{view.Space}-{view.ExternalId}");
+        }
```

Comment:
This feels very wrong. Now you are creating a new metrics object every time you call this endpoint

Reply:
5509a7b6ad94e8dcce9d8b9d3bda5137a9e87dd6 should address that